__tests__/components/Report/ReportForm.test.tsx

```diff
@@ -7,50 +7,74 @@ import { DocFilter } from '@/utils/index';
 import '@testing-library/jest-dom';
 import { act, fireEvent, render, screen } from '@testing-library/react';
 
+jest.mock('react-query', () => ({
+    useQuery: jest.fn(() => ({
+        data: {
+            clients: [],
+        },
+        isLoading: false,
+        refetch: jest.fn(),
+    })),
+
+    useMutation: jest.fn(() => ({
+        data: {
+            clients: [],
+        },
+        isLoading: false,
+        refetch: jest.fn(),
+    })),
+
+    useQueryClient: jest.fn().mockReturnValue({
+        resetQueries: jest.fn(),
+    }),
+}));
+
 describe('Client Info Form Component', () => {
     it('renders correctly with default date range data', async () => {
         await act(async () => {
             render(<ReportForm onSubmit={() => {}} />);
         });
-    
+
         const start = toDateString(
             new Date(new Date().getTime() - DEFAULT_DATE_RANGE * DAY_IN_MS)
         );
         const end = toDateString(new Date());
 
         expect(screen.queryByText('Start Date')).toBeInTheDocument();
         expect(screen.queryByText('End Date')).toBeInTheDocument();
-        expect(screen.queryByText('Submit')).toBeInTheDocument();
-        expect(screen.queryByText('Clear')).toBeInTheDocument();
+        // expect(screen.queryByText('Submit')).toBeInTheDocument();
```

Comment:
If this and the commented code below is deprecated let's remove it instead of leaving it commented